docs/reference/errors-and-warnings/NU3003.md

```diff
@@ -40,4 +40,8 @@ NuGet client tried to verify a package which contains an invalid package signatu
 
 Please file an issue at [NuGet/Home](https://github.com/NuGet/Home/issues) along with the package that generated this problem and the platform on which the issue was encountered.
 
+## Note
```

Comment:
**This applies to notes for all the pages where there is a note in this PR**
Note should be like this:
```suggestion
> [!Note]
> When NuGet’s [signature validation mode](https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages#configure-package-signature-requirements) is set to accept (default), NU3003 is raised as a warning.
> When NuGet’s signature validation mode is set to require, or when running the `nuget verify -signatures` command, NU3003 is elevated from a warning to an error. 
```